apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -57,13 +50,33 @@ router.get(
     }
 
     const instances = await queryRows(
-      sql.select_lti13_instances,
+      sql.select_combined_lti13_instances,
       {
         course_instance_id: res.locals.course_instance.id,
       },
       Lti13CombinedInstanceSchema,
     );
 
+    if (instances.length === 0) {
+      // See if we have configurations per institution
+      const lti_instances = await queryRows(
+        sql.select_lti13_instances,
+        {
+          institution_id: res.locals.institution.id,
+        },
+        Lti13InstanceSchema,
+      );
+
+      res.send(
+        InstructorInstanceAdminLti13NoInstances({
+          resLocals: res.locals,
+          ltiInstances: lti_instances,
```

Comment:
We typically do not rename these variables:
```suggestion
          lti13_instances,
```

Reply:
Fixed in 11fd31e589